src/core/software/pupil-player.md

```diff
@@ -147,26 +147,26 @@ This plugin is **unique**, therefore you can only load one instance of this plug
 ### Analysis Plugins
 These plugins are simple unique plugins, that operate on the gaze data for analysis and visualizations.
 
-#### Offline Surface Tracker
-This plugin is an offline version of the [Surface Tracking](/core/software/pupil-capture/#surface-tracking) plugin for Pupil Capture.
+#### Surface Tracker
+This plugin is an post-hoc version of the [Surface Tracking](/core/software/pupil-capture/#surface-tracking) plugin for Pupil Capture.
 You can use this plugin to detect markers in the recording, define surfaces, edit surfaces, and create and export visualizations of gaze data within the defined surfaces.
 
 <div class="pb-4">
   <img src="../../media/core/imgs/offline-srf-tracker.jpg" style="display:flex;margin:0 auto;">
 </div>
 
-Here is an example workflow for using the `Offline Surface Detector` plugin to generate heatmap visualizations and export surface data reports:
+Here is an example workflow for using the `Surface Tracker` plugin to generate heatmap visualizations and export surface data reports:
 
-  + Load `Offline Surface Detector` plugin - if you already have surfaces defined, the load may take a few seconds because the plugin will look through the entire video and cache the detected surfaces.
+  + Load `Surface Tracker` plugin - if you already have surfaces defined, the load may take a few seconds because the plugin will look through the entire video and cache the detected surfaces.
```

Comment:
```suggestion
  + Load the `Surface Tracker` plugin - if you already have surfaces defined, the load may take a few seconds because the plugin will look through the entire video and cache the detected surfaces.
```